crypto/compactcert/builder_test.go

```diff
@@ -121,10 +121,9 @@ func TestBuildVerify(t *testing.T) {
 	}
 
 	for i := 0; i < npart; i++ {
-		err = b.Add(uint64(i), sigs[i], !doLargeTest)
-		if err != nil {
-			t.Error(err)
-		}
+		a.False(b.Present(uint64(i)))
+		a.NoError(b.IsValid(uint64(i), sigs[i], !doLargeTest))
+		b.Add(uint64(i), sigs[i])
```

Comment:
let's make sure that Add didn't return ad error + there are several instances for that check